crates/nix_rs/src/info.rs

```diff
@@ -10,6 +11,7 @@ pub struct NixInfo {
     pub nix_version: NixVersion,
     pub nix_config: NixConfig,
     pub nix_env: NixEnv,
+    pub group_info: String,
```

Comment:
The user's group should reside in the same struct that contains the user's username, viz. `NixEnv`:

https://github.com/juspay/nix-browser/blob/83746cc2e0e720b1f38f6951ae0fa89c98c1e4fb/crates/nix_rs/src/env.rs#L11-L13

Reply:
Also, the type should be a list not a string, viz.:

```rust
pub struct NixEnv {
  pub user_groups: Vec<String>
```